### PR TITLE
make older testlib compatible with new test structure

### PR DESCRIPTION
**Pull request by jscotka**
I had own copy of testlib prepated to be used with new tests, but after last commit with big amount of changes I found that is was not so good idea.
So the most important change is handling with topdir and also try except of testvm which is not user in new tests.

### CODE REVIEW

File: test/sizzle.v2.1.0.js [hunk 1]
```diff
@@ -0,0 +1 @@
+../lib/sizzle.v2.1.0.js
```
Comment (stefwalter) [inline test/sizzle.v2.1.0.js]: Suggest naming this link test/sizzle.js (without the version number)... similar to how we do with other lib/ dependencies.

File: test/testlib.py [hunk 2]
```diff
@@ -122,10 +125,10 @@ def tryopen(hard=False):
 
     def init_after_load(self):
         # Prevent sizzle from registering with AMD loader, and also claiming the usual global name
-        with open("%s/lib/sizzle.v2.1.0.js" % topdir) as file:
+        with open("%s/sizzle.v2.1.0.js" % topdir) as file:
```
Comment (stefwalter) [inline test/testlib.py]: See above.

**Comment by stefwalter** [comment]
Looks good. I'd just suggest one minor change.

File: test/sizzle.v2.1.0.js (hunk 1 above)
Comment (jscotka) [inline test/sizzle.v2.1.0.js]: Yes, I can change it. It will be better

**Pull request closed without merging**